grpc/request.go

```diff
@@ -0,0 +1,54 @@
+package grpc
+
+import (
+	"errors"
+
+	"github.com/gojek/fiber"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/proto"
+)
+
+type Request struct {
+	Metadata       metadata.MD
+	RequestPayload proto.Message
+	ResponseProto  proto.Message
```

Comment:
It's also strange to store `ResponseProto` within `Request` . Shall it be part of dispatcher as well?

Reply:
yea make sense, since we are going to fix one dispatcher to one hostport and servicemethod, the response can be fixed with dispatcher as well

Reply:
How will the response proto be supplied ? Is it supplied as a file during initialization? https://pkg.go.dev/google.golang.org/protobuf@v1.28.1/types/descriptorpb#FileDescriptorProto

Reply:
Still trying to figure this out, tried this (from server reflection) but the marshalling back into proto doesn't work.